applications/electron/webpack.config.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * This file can be edited to customize webpack configuration.
+ * To reset delete this file and rerun theia build again.
+ */
+// @ts-check
+const config = require('./gen-webpack.config.js');
+
+/**
+ * Expose bundled modules on window.theia.moduleName namespace, e.g.
+ * window['theia']['@theia/core/lib/common/uri'].
+ * Such syntax can be used by external code, for instance, for testing.
+config.module.rules.push({
+    test: /\.js$/,
+    loader: require.resolve('@theia/application-manager/lib/expose-loader')
+}); */
+
+module.exports = config;
```

Comment:
this looks like a generated file or import from the core repo, shouldn't the newline be added there?

Reply:
@duncdrum good point, specially if the file is regenerated without the newline, e.g. every build. Then it's just a matter of time before it's checked-in in that form.

Reply:
so since its a generated file from the main repo, i haven't played with the `newline` as it ll just be overwritten upon regenerating it. @vince-fugnitto maybe you can open an issue in the main repo if the `newline` is important to include